chart/kubeapps/values.yaml

```diff
@@ -1638,6 +1638,8 @@ kubeappsapis:
           ## @param kubeappsapis.pluginConfig.kappController.packages.v1alpha1.defaultUpgradePolicy Default upgrade policy generating version constraints
           ## enum: [ "major", "minor", "patch", "none" ]
           defaultUpgradePolicy: none
+          ## @param kubeappsapis.pluginConfig.kappController.packages.v1alpha1.defaultIncludePrereleases Default policy for allowing pre-releases to be installed
+          defaultIncludePrereleases: false
```

Comment:
Sorry, just a thought: I know I'd suggested a boolean when reviewing your other PR - but it might be just as simple to actually have a value here (ie. rename `defaultPrereleasesVersionSelection` that is empty (nil) by default, but can be set to `{}` or whatever? This way, we don't have to later change this config option to support actually setting version selections for pre releases? I'm not suggesting we implement support for them yet, it'd still effectively be a boolean option, but one that doesn't need to change later.

Fine either way.

Reply:
No, it's fine. In fact, after re-reading the carvel issues and docs I've started having some doubts/questions, so I asked them at: https://kubernetes.slack.com/archives/CH8KCCKA5/p1643376802571959

In short, `prereleases` can have three values:

1. `prereleases: {}`: allow any prerelease if the version constraint allows it.
2. `prereleases: nil`: disallow any prerelease 
    * Exception: if the version constraint is a prerelease itself:
          1. Current behavior: error, it won't install a prerelease if `prereleases: nil`:
          2. Future behavior: install it, it won't be required to set `prereleases:  {}` if the version constraint is a prerelease
3. `prereleases: {Identifiers: []string{"rc"}}`: allow only prerelease with "rc" as part of the name if the version constraint allows it.

> I'm not suggesting we implement support for them yet, it'd still effectively be a boolean option for now, but one that doesn't need to change later.

It is mostly the same effort, I mean, it is just a matter of appending strings or not. I'll update this PR with the full implementation soon. Thanks for the suggestion, much better this way!

Reply:
Also, I'd like to add (in a separate PR) another default config like `allowDowngrading` or sth, to add the `packaging.carvel.dev/downgradable: "` annotation to the PackageInstall objs to allow them to be downgraded: https://carvel.dev/kapp-controller/docs/latest/package-consumer-concepts/#downgrading

Reply:
Nice, yes, that'd be useful (as a kubeapps config default).

Reply:
Done: https://github.com/kubeapps/kubeapps/pull/4192  :)